Source/StdoutLogger.cpp

```diff
@@ -1,56 +1,83 @@
 #include <AndroidExtensions/StdoutLogger.h>
-#include <unistd.h>
 #include <android/log.h>
 #include <pthread.h>
 #include <stdio.h>
+#include <stdlib.h>
+#include <unistd.h>
 
 namespace
 {
-    static int pfd[2];
-    static int fd_saved[2];
-}
+    int fd_stdout[2];
+    int fd_stderr[2];
+    int fd_saved[2];
```

Comment:
I'd keep these as static to not polute linker. fd_saved is left over?

Reply:
It does not pollute linker. It's in an anonymous namespace. We don't want to put `static` and anonymous namespace. That's not best practice.

Good point on `fd_saved`.